backend/lrclib.go

```diff
@@ -14,6 +19,38 @@ import (
 	"github.com/dweymouth/supersonic/backend/mediaprovider"
 )
 
+func FetchLrcLibLyricsCached(name, artist, album string, durationSecs int, cacheDir string) (*mediaprovider.Lyrics, error) {
+	hash := makeTrackIdHash(name, artist, album)
+	cacheFilePath := filepath.Join(cacheDir, fmt.Sprintf("%s_lyrics.txt", hash))
```

Comment:
Let's use a subdirectory "lrclib" to store the cached LrcLib lyrics to make it easier to manually clear, and later write a clean task for, etc.